src/ucp/proto/proto_common.h

```diff
@@ -94,9 +94,14 @@ typedef struct {
 
     /* Offset in uct_iface_attr_t structure of the field which specifies the
      * maximal number of iov elements for the UCT operation used by this
-    * protocol */
+     * protocol */
     ptrdiff_t               max_iov_offs;
 
+    /* Offset in uct_iface_attr_t structure of the field which specifies the
+     * optimal alignment for buffer address for the UCT operation used
+     * by this protocol */
+    ptrdiff_t               opt_align_offs;
```

Comment:
maybe we can move this to ucp_proto_multi_init_params_t, since only multi-send protocol can actually do something to fix the alignment